src/composer.json

```diff
@@ -17,17 +17,17 @@
         "ext-zlib": "*",
         "ext-PDO": "*",
         "ext-mbstring" : "*",
-        "pimple/pimple": "v3.3.0",
+        "pimple/pimple": "^3.3.1",
```

Comment:
I have resolved all your required dependencies for your pull request to pass the tests. composer.json and composer.lock should function in current state.